ts/features/bonus/cgn/store/reducers/eyca/activation.ts

```diff
@@ -47,4 +54,18 @@ export const eycaActivationStatusSelector = (
   state: GlobalState
 ): EycaActivationState => state.bonus.cgn.eyca.activation;
 
+// return the cgn eyca status
+// TODO Use this selector in PR https://github.com/pagopa/io-app/pull/2872
+//  to check the EYCA activation status is not ERROR
+export const cgnEycaActivationStatus = createSelector(
+  eycaActivationStatusSelector,
+  (activation: EycaActivationState): CgnEycaActivationStatus | undefined =>
+    isReady(activation) ? activation.value : undefined
```

Comment:
```suggestion
    getValueOrElse(activation,undefined)
```
same but more coincise

Reply:
Fixed in 059777b